src/core/server/base.js

```diff
@@ -42,6 +42,7 @@ import {
   langToLocale,
   makeI18n,
 } from 'core/i18n/utils';
+import { setTaarId } from 'disco/reducers/discoResults';
```

Comment:
Do we really want to set a TAAR `id` in the `disco` results? I'd create a new reducer for that. Also, we've used `clientId` (or client ID or telemetry client ID) for a while, so I would not change that name, because it is a bit confusing. Sure the cookie name in FF is `taarId` but that's just the cookie name. We can have `clientId` on our side.

I would create a `telemetry` reducer maybe and store the `clientId` in it. Because the content is hashed, maybe `hashedClientId` would be a better name and we would have an action creator named `setHashedClientId` for example.

Reply:
That could be done in a PR before this one so that we would have one issue explaining why/how we plan to fully tackle #7218 at least (because understanding the aim of this PR was tough 😞)

Reply:
> Do we really want to set a TAAR id in the disco results? I'd create a new reducer for that. Also, we've used clientId (or client ID or telemetry client ID) for a while, so I would not change that name, because it is a bit confusing. Sure the cookie name in FF is taarId but that's just the cookie name. We can have clientId on our side.

I thought about this (not using in the disco reducer) but I ultimately decided ti leave it here. I can definitely move it and do this in another PR like you suggest.

> That could be done in a PR before this one so that we would have one issue explaining why/how we plan to fully tackle #7218 at least (because understanding the aim of this PR was tough 😞)

Sorry, I should have put in a description above 🙈.